.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: maven
+    directory: "/"
+    labels:
+      - dependencies
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 99
```

Comment:
wb 10?

Reply:
it doesn't matter but ok.

Reply:
> wb 10?

What's the point of limiting it?

Reply:
> > wb 10?
> 
> What's the point of limiting it?

treasury won't have _that_ much dependencies, and I personally don't want the PR tab fulfilled top to bottom with PRs from dependabot.

Reply:
Oh, I see

Reply:
Now I understand what you ment